PowerEditor/src/Notepad_plus.cpp

```diff
@@ -6145,7 +6145,6 @@ Quote quotes[nbQuote] =
 	{"Anonymous #120", "Life is what happens to you while you're looking at your smartphone."},
 	//{"Anonymous #121", ""},
 	{"Anonymous #122", "Nerd?\nI prefer the term \"Intellectual badass\"."},
-	{"Anonymous #123", "How can you face your problem if your problem is your face?"},

```

Comment:
Instead of simply removing the line change it to an out-commented line as e.g. "Anonymous 121".